build_defs.bzl

```
@@ -69,7 +82,14 @@ def _compile_circuit_impl(ctx):
     name = _c_name(src.basename[:-len(".circom")])
     outputs = [
         name + ".r1cs",
-    ] + _c_srcs(name) + _c_data(name)
+    ]
+    if ctx.attr.generate_c:
+        outputs.append(_c_srcs(name))
+        outputs.append(_c_data(name))
```

This change is needed because both `c_srcs()` and `c_data()` already return a list.

```suggestion
        outputs += _c_srcs(name)
        outputs += _c_data(name)
```